app/controllers/page_views_controller.rb

```diff
@@ -28,7 +30,7 @@ def update
   private
 
   def update_article_page_views
-    return if Rails.env.production? && rand(8) != 1 # We don't need to update the article page views every time.
+    return if Rails.env.production? && rand(10) != 1 # We don't need to update the article page views every time.
```

Comment:
Just a thought on making performance comparisons. Should we leave these numbers(also update_organic_page_views below) as is so that we can get a clear picture of the performance impact of removing the application controller? Otherwise, we have two variables changes and if we get a performance boost it will be hard to tell which one it was from.

Reply:
That makes sense